src/index.html

```diff
@@ -7,6 +7,70 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="calendar main__calendar calendar--month-length_29 calendar--start_wen">
+        <div class="calendar__day day-1"></div>
+
+        <div class="calendar__day day-2"></div>
+
+        <div class="calendar__day day-3"></div>
+
+        <div class="calendar__day day-4"></div>
+
+        <div class="calendar__day day-5"></div>
+
```

Comment:
```suggestion
        <div class="calendar__day"></div>

        <div class="calendar__day"></div>

        <div class="calendar__day"></div>

        <div class="calendar__day"></div>

        <div class="calendar__day"></div>

```
Remove this class everywhre use `nth-child` instead